runtime/test/planificator-test.js

```diff
@@ -417,4 +417,13 @@ describe('Planificator', function() {
     assert.isEmpty(planificator._current.plans);
     assert.equal(4, planChanged);
   });
+  it('cancels planning', function() {
```

Comment:
Perhaps add a test for search changing cancelling planning underway as well?

Reply:
done